demo/NostrSDKDemo/Demo Views/QueryRelayDemoView.swift

```diff
@@ -97,7 +97,7 @@ struct QueryRelayDemoView: View {
         } else {
             authors = [authorPubkey]
         }
-        return Filter(authors: authors, kinds: [selectedKind])
+        return Filter(authors: authors, kinds: [selectedKind])!
```

Comment:
Unrelated fix for the demo app.

The `Filter` initializer was [changed](https://github.com/nostr-sdk/nostr-sdk-ios/pull/154) to be able to return `nil` if `tags` contains tag names that are not in the basic Latin alphabet of A-Z or a-z. However, we're not using tags, so it will never be `nil`. I recognize that forcing the unwrap of an optional is bad, but since it's just a demo app and we know that it will always be non-nil, I thought it would be ok.

Reply:
This change is fine, but it pollutes this PR and the resulting commit. If we were ever to need to revert the PR's squashed commit, it would revert this unrelated thing also and we might not notice it.

Reply:
Fair point. I got lazy. I'm happy to split this out into a separate PR.